omod/src/main/java/org/openmrs/web/servlet/LoginServlet.java

```diff
@@ -178,7 +180,14 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			catch (ContextAuthenticationException e) {
 				// set the error message for the user telling them
 				// to try again
-				httpSession.setAttribute(WebConstants.OPENMRS_ERROR_ATTR, "auth.password.invalid");
+				Integer maximumAlowedAttempts = 7;
```

Comment:
Why did you make the maximum allowed attempts 7?

Reply:
because thats the default maximum allowed attemptts

Reply:
i have realised its a configurable figure , so let just try to acces it using a global property